src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -40,161 +41,167 @@ type TemplatesCatalogDrawerCreateFormProps = {
-            setIsQuickCreating(false);
-            setQuickCreateError(err);
-          });
-      };
+        .catch((err) => {
+          setIsQuickCreating(false);
+          setQuickCreateError(err);
+        });
+    };
 
-      const onCustomize = (e) => {
-        e.preventDefault();
-        let catalogUrl = `templatescatalog/customize?name=${template.metadata.name}&namespace=${template.metadata.namespace}&defaultSourceExists=${canQuickCreate}`;
+    const onCustomize = (e) => {
+      e.preventDefault();
+      let catalogUrl = `templatescatalog/customize?name=${template.metadata.name}&namespace=${template.metadata.namespace}&defaultSourceExists=${canQuickCreate}`;
 
-        if (vmName) {
-          catalogUrl += `&vmName=${vmName}`;
-        }
+      if (vmName) {
+        catalogUrl += `&vmName=${vmName}`;
+      }
 
-        history.push(catalogUrl);
-      };
+      history.push(catalogUrl);
+    };
 
-      return (
-        <form className="template-catalog-drawer-form" id="quick-create-form">
-          <Stack hasGutter>
-            {canQuickCreate ? (
-              <>
-                <StackItem>
-                  <Split hasGutter>
-                    <SplitItem>
-                      <FormGroup
-                        label={t('VirtualMachine name')}
-                        isRequired
-                        className="template-catalog-drawer-form-name"
-                        fieldId="vm-name-field"
-                      >
-                        <TextInput
-                          isRequired
-                          type="text"
-                          data-test-id="template-catalog-vm-name-input"
-                          name="vmname"
-                          aria-label="virtualmachine name"
-                          value={vmName}
-                          onChange={setVMName}
-                        />
-                      </FormGroup>
-                    </SplitItem>
-                    <SplitItem>
-                      <DescriptionList>
-                        <DescriptionListGroup>
-                          <DescriptionListTerm>{t('Project')}</DescriptionListTerm>
-                          <DescriptionListDescription>{namespace}</DescriptionListDescription>
-                        </DescriptionListGroup>
-                      </DescriptionList>
-                    </SplitItem>
-                  </Split>
-                </StackItem>
-                <StackItem />
-                <StackItem>
-                  <Checkbox
-                    id="start-after-create-checkbox"
-                    isChecked={startVM}
-                    onChange={(v) => setStartVM(v)}
-                    label={t('Start this VirtualMachine after creation')}
-                  />
-                </StackItem>
-              </>
-            ) : (
+    useEffect(() => {
+      updateTabsData((tabsDataDraft) => {
+        tabsDataDraft.startVM = startVM;
+      });
+    }, [startVM, updateTabsData]);
```

Comment:
I would take this updateTabsData method and use it in the Checkbox component in the onChange prop something like this:
```
onChange={(checked) => {
  setStartVM(checked);
  updateTabsData((currentTabsData) => {
    return { ...currentTabsData, startVM: checked };
  });
}}
```